EntityDumpLog.txt

```diff
@@ -0,0 +1,57 @@
+ENTITY POSITION DUMP
```

Comment:
This file should not be committed to the repo
Add .txt files to the gitignore and remove this file from your branch